src/pages/release-notes.md

```diff
@@ -47,6 +49,12 @@ By default, the versions listed in the [Tractus-X changelog](https://eclipse-tra
 [Latest] – latest released version  
 [Recommended] – recommended by the Catena-X association (Compliant with the [Catena-X Standards](/docs/next/standards/overview) and [End-to-End](docs/next/working-model/release-management/development-and-release/e2e-test) tested)
```

Comment:
In my understanding, there is no formal check whether a component is actually compliant to a standard or not?

Reply:
Hey @jSchuetz88,

if there is an idea or proposal on a formal check happy to introduce it immediately. As of now, we would have gone with a "confidence check / official statement" from the respective Expert Group/Comittee.

So they are responsible for it.